components/list/list-item-drag-drop-mixin.js

```diff
@@ -555,9 +555,14 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 		const selectionInfo = rootList.getSelectionInfo(rootList.dragMultiple);
 
 		if (rootList.dragMultiple && selectionInfo.keys.length > 1) {
-			const dragImage = document.createElement('d2l-list-item-drag-image');
-			dragImage.count = selectionInfo.keys.length;
-			this.shadowRoot.appendChild(dragImage);
+			let dragImage = this.shadowRoot.querySelector('d2l-list-item-drag-image');
+			if (dragImage) {
+				dragImage.count = selectionInfo.keys.length;
+			} else {
+				dragImage = document.createElement('d2l-list-item-drag-image');
+				dragImage.count = selectionInfo.keys.length;
+				this.shadowRoot.appendChild(dragImage);
+			}
```

Comment:
Nit: to remove duplicate code:

```suggestion
			let dragImage = this.shadowRoot.querySelector('d2l-list-item-drag-image');
			if (!dragImage) {
				dragImage = document.createElement('d2l-list-item-drag-image');
				this.shadowRoot.appendChild(dragImage);
			}
			dragImage.count = selectionInfo.keys.length;
```